kube-psp-advisor.go

```diff
@@ -53,7 +53,7 @@ func inspectPsp(kubeconfig string, namespace string, withReport, withGrant bool)
 	return nil
 }
 
-func convertPsp(podObjFilename string, pspFilename string) error {
+func convertPsp(podObjFilename string, pspFilename string, defaultPspFile string) error {
```

Comment:
can we keep the compatibility of function `convertPsp`? maybe you can create a new function.

Reply:
yes. I will try change prototype :)

Reply:
@Kaizhe Thanks for reviewing 👍 
I tried to save compatibility and pushed new commit. Please check it.